docassemble_webapp/docassemble/webapp/server.py

```diff
@@ -2844,7 +2844,7 @@ def progress_bar(progress, interview):
         percentage = str(int(progress)) + '%'
     else:
         percentage = ''
-    return '<div class="progress mt-2"><div class="progress-bar" role="progressbar" aria-valuenow="' + str(progress) + '" aria-valuemin="0" aria-valuemax="100" style="width: ' + str(progress) + '%;">' + percentage + '</div></div>\n'
+    return '<div class="progress mt-2"><div class="progress-bar" aria-label="Interview Progress" role="progressbar" aria-valuenow="' + str(progress) + '" aria-valuemin="0" aria-valuemax="100" style="width: ' + str(progress) + '%;">' + percentage + '</div></div>\n'
```

Comment:
I'll fix this when I merge the PR, but note that this introduces hard-coded English. I changed it to `aria-label="' + noquote(word('Interview Progress')) + '"`

Reply:
Thanks, sorry I keep missing those!